src/main/frontend/components/block.cljs

```diff
@@ -1635,43 +1635,36 @@
        (not (d/has-class? (gobj/get e "target") "bullet"))
        (not @*dragging?)))
 
+(rum/defc breadcrumb-fragment
+  [href label]
+  [:a {:href href} (apply str label)])
```

Comment:
The block title (label here) might contain bold or other formatted text, `(apply str label)` here will remove the formats.
![CleanShot 2021-06-29 at 19 57 11](https://user-images.githubusercontent.com/479169/123793533-63b03e00-d914-11eb-90e9-04523e2d999c.png)

Reply:
Done. I checked other similar places and it looks like I need to wrap the result of `map-inline` with `->elem`.